RestImageResize.EpiServer/readme.txt

```diff
@@ -0,0 +1,12 @@
+﻿RestImageResize for EPiServer
```

Comment:
One is repo readme, another one for NuGet package to be displayed after installing it